modules/virtue-admin/src/main/resources/templates/port-forwarding.tpl

```diff
@@ -15,5 +18,10 @@ portForward () {
         ssh -i $CERT $USER@$GUESTIP "echo $INTERNALPORT=$EXTERNALPORT >> $REMOTEPORTFILE"
 }
 
-
-portForward ${externalPort} ${guestIp} ${internalPort}
+for (( i=0; i<$NUM_PORTS; i++ ))
+do
+	extPort=$((i + ${externalPort}))
+	intPort=$((i + ${internalPort}))
+	echo "i=$i Port $extPort -> $intPort"
+	portForward $extPort ${guestIp} $intPort

```

Comment:
I was curious so I looked in the script. You *could* collect the output from the portForward calls for the REMOTEPORTFILE and do one ssh command to create that file. I'm not sure if it's worth it, though. What's that file for, anyway?

Reply:
That is the file that the sensors use to know what host and port they are from.